nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -46,6 +50,13 @@
  */
 public class Document extends NuxeoEntity {
 
+    @JsonIgnore
```

Comment:
You're right. Statics are implicitly transient, so I don't need to declare them as such. Serialization is for serializing instances, not classes. Static fields will be reinitialized to whatever value they are set to when the class is loaded. So `@JsonIgnore` is not really needed here. References from Stack Overflow:

- [Java static serialization rules?](http://stackoverflow.com/questions/6429462/java-static-serialization-rules)
- [Is @JsonIgnore required for ignoring static field?](http://stackoverflow.com/questions/42123971/is-jsonignore-required-for-ignoring-static-field)